repoSets.json

```diff
@@ -570,11 +575,6 @@
         "repo": "CORS",
         "inclusionLevel": "AllItems"
       },
-      {
-        "org": "aspnet",
-        "repo": "DependencyInjection",
-        "inclusionLevel": "AllItems"
```

Comment:
Similar to the above comment, we don't want to remove the repo entirely, but instead just flip it from AllItems to ItemsAssigned.